composer.json

```diff
@@ -9,7 +9,7 @@
     ],
     "minimum-stability": "stable",
     "require-dev": {
-        "phpunit/phpunit": "3.7.*",
+        "phpunit/phpunit": "4.6.*",
```

Comment:
@GrahamCampbell My bad, 4.5 is correct. Will fix after reply.

For the second, Travis uses own internal PHPUnit. This is probably for local use.